src/ApiCode.ts

```diff
@@ -1,48 +1,65 @@
-async function stopIDToFirst5Buses(stopID : string): Promise<JSON[]> {
+async function stopIDToFirst5Buses(stopID : string): Promise<Bus[]> {
     const  myUrl : string = "https://api.tfl.gov.uk/StopPoint/" + stopID + "/Arrivals";
     const response = await fetch(myUrl);
     const data = await response.json();
     // deal with JSON response
     const busesArray = []
     const numOfBuses = Math.min(5, data.length)
     for (let i = 0 ; i < numOfBuses;  i ++){
-        busesArray.push(data[i])
+        busesArray.push({lineName : data[i].lineName, timeToStation: data[i].timeToStation})
     }
     return busesArray
 
 
   }
 
+  export interface Bus {
+    lineName: string
+    timeToStation: string
+  }
+
+
+  export interface BusStop {
+   
+    stationName: string;
+    buses : Bus[]
+  
+}
```

Comment:
Logically, the `Bus` and `BusStop` interfaces are used across the app are aren't tied to the `ApiCode` alone. They should be pulled out into their own files, maybe in a `interfaces` folder.